lms/models/application_instance.py

```diff
@@ -38,6 +41,24 @@ class ApplicationInstance(BASE):
         nullable=False,
     )
 
+    # LMS info linked to this consumer_key, description of each one from the spec
+    # https://www.imsglobal.org/specs/ltiv1p1p1/implementation-guide
+
+    # This is a unique identifier for the TC.  A common practice is to use the DNS of the organization or the DNS of the TC instance.
+    tool_consumer_instance_guid = sa.Column(sa.String, nullable=True)
+
+    # In order to better assist tools in using extensions and also making their user interface fit into the TC's user interface that they are being called from
+    tool_consumer_info_product_family_code = sa.Column(sa.String, nullable=True)
+
+    # This is a plain text user visible field – it should be about the length of a line
+    tool_consumer_instance_description = sa.Column(sa.String, nullable=True)
+
+    # This is the URL of the consumer instance
```

Comment:
Copied the comments from the spec, this one it's not great but left it for consistency